README.md

```diff
@@ -310,14 +310,70 @@ $ ninja -C out/Release
 
 ### Running tests
 
-Run:
+#### Build
+
+Build (includes tests):
 
 ```bash
 $ ./gyp_uv.py -f make
 $ make -C out
+```
+
+#### Run all tests
+
+```bash
 $ ./out/Debug/run-tests
 ```
 
+#### Run one test
+
+The list of all tests is in `test/test-list.h`.
+
+This invocation will cause the `run-tests` driver to fork and execute `TEST_NAME` in a child process:
+
+```bash
+$ ./out/Debug/run-tests TEST_NAME
+```
+
+This invocation will cause the `run-tests` driver to execute the test within the `run-tests` process:
+
+```bash
+$ ./out/Debug/run-tests TEST_NAME TEST_NAME
+```
+
+#### Debugging tools
+
+If making use of debugging tools like gdb, valgrind, perf etc.:
+
+When running the test from within the `run-tests` process (`run-tests TEST_NAME TEST_NAME`), tools like gdb and valgrind work normally.
+When running the test from a child of the `run-tests` process (`run-tests TEST_NAME`), use these tools in a fork-aware manner.
+
+##### Fork-aware gdb
```

Comment:
One thing I'm noticing from looking at the rendered page is that these `<h5>` headers seem smaller than the paragraph text.

Reply:
Indeed. I could try instead with a table, like

| Debugging tool | Test invocation | Tips and tricks |
|---------------------|---------------------|--------------------|
| gdb                   | `run-tests TEST_NAME TEST_NAME` | Use as normal |
| gdb                   | `run-tests TEST_NAME` | Use `follow-fork-mode` |

and so on.

Reply:
I'm not sure if a table would work well because, for example, the `valgrind` command is pretty long. I don't really have a better idea, so I might just leave it up to someone else 😄